.github/workflows/aws-close-stale-issues.yml

```diff
@@ -0,0 +1,62 @@
+name: "Close Stale Issues"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every day at midnight
+  - cron: "0 0 * * *"
+
+# This job currently has two workflows
+# 1. Stale issue workflow
+# - If an issue has the `response-requested-label` for `days-before-stale`
+#   then add the `stale-issue-label` and comment with the `stale-issue-message`
+# - If an issue has the `stale-issue-label` for `days-before-close` then close the issue
+# 2. Ancient issue workflow (currently disabled)
+# - If an issue has been open for `days-before-ancient` then add then `ancient-issue-message`
+#   and add the `stale-issue-label` (which starts the Stale issue workflow)
+jobs:
+  cleanup:
+    # this workflow will always fail in forks; bail if this isn't running in the upstream
+    if: github.repository == 'pulumi/pulumi-aws'
+    permissions:
+      issues: write
+      contents: read
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v6
+      with:
+        # Types of issues that will be processed
+        issue-types: issues
+
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        stale-issue-message: This issue has not received a response in a while. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+
+        # TODO: turn on ancient-issue workflow once we are ready
+        # ancient-issue-message: This issue has not received any attention in 1 year. If you want to keep this issue open, please leave a comment below and auto-close will be canceled.
+
+        # These labels are required
+        stale-issue-label: closing-soon
+        exempt-issue-labels: no-autoclose
```

Comment:
Should this include `p1`, `customer-feedback`, `impact/regression`.

Per your proposal it sounds like it should also be `awaiting-upstream` as we wanted to have an automated check there for whether the upstream has been closed or not to gate staling it out?

Reply:
This only affects issues labeled with `awaiting-feedback` which I think should affect all issues regardless of label. We can always add the `no-autoclose` to issues that we require a response on, but want to leave open even if we never get a response (would we ever want that?)